content/en/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Kubernetes operators
+reviewers:
+- erictune
+- lavalamp
+- cheftako
+- chenopis
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Operators are software extensions to Kubernetes that make use of third-party
+resources to manage applications and their components. Operators follow
+Kubernetes priciples, notably the [control loop](/docs/concepts/#kubernetes-control-plane).
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Motivation
+
+The Operator concept aims to capture the key aim of a human operator who
+is managing a service or set of services. Human operators who look after
+specific applications and services have deep knowledge of how the system
+ought to behave, how to deploy it, and how to react if there are problems.
+
+People who run workloads on Kubernetes often like to use automation to take
+care of repeatable tasks. The Operator pattern captures how you can write
+code to automate a task beyond what Kubernetes itself provides.
+
+## Operators in Kubernetes
+
+Kubernetes is designed for automation. Out of the box, you get lots of
+built-in automation from the core of Kubernetes. You can use Kubernetes
+to automate deploying and running workloads, *and* you can automate how
+Kubernetes does that.
+
+Kubernetes' {{< glossary_tooltip text="controllers" term_id="controller" >}}
+concept lets you extend the cluster's behaviour without modifying the code
+of Kubernetes itself.
+Operators are clients of the Kubernetes API that act as controllers for
+a [Custom Resource](/docs/concepts/api-extension/custom-resources/).
+
+## Deploying Operators
+
+The most common way to deploy Operators is to add them to your cluster
+running outside of the {{< glossary_tooltip text="control plane" term_id="control-plane" >}},
+much as you would run any containerized application.
+For example, you can run the controller in your cluster as a Deployment.
+
+Just as with any Controller, you can also run an Operator entirely outside
+of Kubernetes, and sometimes that makes sense. For example, you might want to
+do this if your Operator is involved with ensuring the availability of Nodes
+themselves.
+
+## Using an Operator {#using-operators}
+
+Once you have an Operator deployed, you'd use it by adding, modifying or
+deleting the kind of resource that the Operator uses. For example:
+
+```shell
+kubectl get BackupSchedule         # for a fictional custom resource named BackupSchedule
+
+kubectl edit BackupSchedule/weekly # manually change the settings
+```
+
+&hellip;and that's it! Assuming the Operator works how you want, your work is done.
+
```

Comment:
not sure about the &hellip;

Reply:
I want to convey simplicity. What are some other ways to do that &mdash especially, ones the line up with the style guide?

(I haven't looked for examples of how other Kubernetes docs handle this, yet).

Reply:
nit: You could just say, `That's it`, or remove the exclamation. Instead, `Assuming the Operator is running or working as designed, ...`

I think this is close enough. I already commented on a few things that could be cleaned up before merging so that there is not more immediate work that needs to be done (such as the extra link to the `What's Next` when the content is a few lines below).
/lgtm

Reply:
Also, you may want to check with sig-api-machinery (controllers, kubebuilder), if you have technical reservations.

Reply:
> Also, you may want to check with sig-api-machinery (controllers, kubebuilder), if you have technical reservations.

I don't actually know how to make that happen. I looked at https://kubernetes.io/docs/contribute/intermediate/#review-a-pr and didn't work out what I ought to do.

Reply:
Have you asked on sig-docs, sig-api-machinery slack channels? I am not entirely certain if sig-api-machinery is the correct group to review. In theory, you should be able to label a PR with, for example, sig/api-machinery.